ansible/files/telemetry.sh

```diff
@@ -142,41 +94,19 @@ source $HOME/.bashrc
 ipython profile create
 cat << EOF > $HOME/.ipython/profile_default/startup/00-pyspark-setup.py
 import os
-
 spark_home = os.environ.get('SPARK_HOME', None)
 execfile(os.path.join(spark_home, 'python/pyspark/shell.py'))
 EOF
 
-# Dump Spark logs to a file
-cat << EOF > $SPARK_HOME/conf/log4j.properties
-# Initialize root logger
-log4j.rootLogger=INFO, FILE
-
-# Set everything to be logged to the console
-log4j.rootCategory=INFO, FILE
-
-# Ignore messages below warning level from Jetty, because it's a bit verbose
-log4j.logger.org.eclipse.jetty=WARN
-
-# Set the appender named FILE to be a File appender
-log4j.appender.FILE=org.apache.log4j.FileAppender
-
-# Change the path to where you want the log file to reside
-log4j.appender.FILE.File=/mnt/spark.log
-
-# Prettify output a bit
-log4j.appender.FILE.layout=org.apache.log4j.PatternLayout
-log4j.appender.FILE.layout.ConversionPattern=%d{yy/MM/dd HH:mm:ss} %p %c{1}: %m%n
-EOF
-
 # Setup plotly
-mkdir $HOME/.plotly && aws s3 cp s3://telemetry-spark-emr/plotly_credentials $HOME/.plotly/.credentials
+mkdir -p $HOME/.plotly && aws s3 cp $TELEMETRY_CONF_BUCKET/plotly_credentials $HOME/.plotly/.credentials
 
 # Setup dotfiles
 $(git clone --recursive https://github.com/vitillo/dotfiles.git;
   cd dotfiles;
-  make install-vim install-tmux install-emacs;)
+  make install-vim install-tmux install-emacs)
 
+# Launch IPython
 mkdir -p $HOME/analyses && cd $HOME/analyses
-wget https://raw.githubusercontent.com/vitillo/emr-bootstrap-spark/master/Telemetry%20Hello%20World.ipynb
+wget -nc https://raw.githubusercontent.com/vitillo/emr-bootstrap-spark/master/Telemetry%20Hello%20World.ipynb
```

Comment:
I actually tried to make it consistent and use only spaces. Where do you still see tabs?